src/dev/build_api_docs/README.md

```diff
@@ -0,0 +1,12 @@
+# Autogenerated API documentation
```

Comment:
Are you open to me converting this directory into a package?

Reply:
Sure!

Reply:
Opened https://github.com/stacey-gammon/kibana/pull/16